src/bittytax/conv/parsers/coinbase.py

```diff
@@ -168,15 +169,31 @@ def _do_parse_coinbase(
             buy_value=total_ccy,
             wallet=WALLET,
         )
-    elif row_dict["Transaction Type"] == "Send":
+    elif row_dict["Transaction Type"] in ("Send", "Withdrawal"):
         data_row.t_record = TransactionOutRecord(
             TrType.WITHDRAWAL,
             data_row.timestamp,
             sell_quantity=Decimal(row_dict["Quantity Transacted"]),
             sell_asset=row_dict["Asset"],
+            sell_value=total_ccy,
+            wallet=WALLET,
+        )
+    elif row_dict["Transaction Type"] == "Buy":
```

Comment:
Trying to understand why you split the handling of "Buy" and "Advanced Trade Buy"?

I don't believe this is related to the new transfers types being added?

Was it because you wanted to add the buy/sell/fee values? If so why are these required, with trades on Coinbase one of the assets is normally fiat already?

Reply:
The buy orders were not converting correctly from my file (missing values) and I didn't have any advanced trades to compare against, so I built the new blocks.